execution/gethexec/node.go

```diff
@@ -147,6 +149,9 @@ func (c *Config) Validate() error {
 	if c.forwardingTarget != "" && c.Sequencer.Enable {
 		return errors.New("ForwardingTarget set and sequencer enabled")
 	}
+	if c.MultigasCollector.OutputDir != "" && c.MultigasCollector.BatchSize <= 0 {
```

Comment:
I don't think this matches the behavior of the command-line flag's documentation.
The flag's help string says that if `BatchSize <= 0` a default value will be used. But, this is going to raise an error.

Reply:
I will change the outdated comment. I prefer this behaviour rather than setting the default value, since an empty OutputDir is used for enabling the collector